### DISCUSSION TITLE
Everything below this plugin call is not possible?

### DISCUSSION THREAD

**Discussion by budimanjojo** [wbthomason/packer.nvim#708] [Q&A] [answered]
Hello, I'm stuck here because everything is not making any sense to me.
I'm transitioning my vim-plug to packer, and I just don't understand what am I doing wrong. I don't know if this is because of how lua work or this is packer thing.
So, I have this in my `plugins.lua` file:
```
local fn = vim.fn
local install_path = fn.stdpath('data')..'/site/pack/packer/start/packer.nvim'
if fn.empty(fn.glob(install_path)) > 0 then
  packer_bootstrap = fn.system({'git', 'clone', '--depth', '1', 'https://github.com/wbthomason/packer.nvim', install_path})
end

vim.cmd[[
  augroup packer_user_config
    autocmd!
    autocmd BufWritePost plugins.lua source init.lua | PackerCompile
  augroup END
]]

return require('packer').startup(function()
    use 'wbthomason/packer.nvim'

  -- Automatically set up your configuration after cloning packer.nvim
  -- Put this at the end after all plugins
  if packer_bootstrap then
    require('packer').sync()
  end
end)
```
Everything is working fine, until I want to move the block:
```
vim.cmd[[
  augroup packer_user_config
    autocmd!
    autocmd BufWritePost plugins.lua source init.lua | PackerCompile
  augroup END
]]
```
to the end of the file. I will get this error: `E5113: Error while calling lua chunk: vim.lua:64: /home/budiman/.config/nvim/lua/plugins.lua:24: '<eof>' expected n
ear 'vim'`

So I try putting anything below the `end)` and I will get the same error. Why can't I put code below that `end)` line?

**Comment by nanotee** [comment, marked as answer]
Lua doesn't allow putting anything after a return statement, you should probably delete the `return` keyword from your `plugins.lua` file

**Reply by budimanjojo** [reply]
I just delete that? does it make things work differently?
